### DISCUSSION TITLE
Performance score differing from a manual lighthouse scan

### DISCUSSION THREAD

**Discussion by LunarDevelopment** [spatie/lighthouse-php#47] [Q&A]
Hello,

I'm getting consistent scores on all metrics except "Performance" which is sometimes differing wildly from a manual lighthouse scan performed in Chrome directly. e.g. scoring 98 from this library vs 30 with chrome lighthouse scan.

This affects both the Mobile and Desktop form factors.

Any idea why this is ?

example domain: simon-taylor.co.uk 

v 1.2.2 of the library used.

**Comment by hansiallen** [comment]
This library doesn't have performance throttling enabled by default. To do this using the throttleCPU and throttleNetwork methods. For more info check out [https://spatie.be/docs/lighthouse-php/v1/usage/configuring-a-run#content-enable-throttling](https://spatie.be/docs/lighthouse-php/v1/usage/configuring-a-run#content-enable-throttling). There may still be slight variations due to differing hardware across devices, and factors that make performance vary over time such as latency and server load. To compensate for system specs, check out [https://spatie.be/docs/lighthouse-php/v1/usage/working-with-results#content-getting-the-benchmark-index](https://spatie.be/docs/lighthouse-php/v1/usage/working-with-results#content-getting-the-benchmark-index) for how to get a benchmark index of your system (and even this isn't that accurate and will vary)
